crates/oxc_linter/src/snapshots/no_named_as_default.snap

```diff
@@ -0,0 +1,34 @@
+---
+source: crates/oxc_linter/src/tester.rs
+assertion_line: 143
+expression: no_named_as_default
+---
+  ⚠ eslint-plugin-import(no-named-as-default): Using exported name "foo" as identifier for default export.
```

Comment:
Do we need to say what the export name is from the other side? The current messages doesn't say about why this is bad.